src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceAPISource.java

```diff
@@ -197,7 +197,7 @@ public ReferenceBases getReferenceBases(final PipelineOptions pipelineOptions, f
               if (received.length < interval.size()) {
                   final List<byte[]> blobs = new ArrayList<>();
                   blobs.add(received);
-                  while (result.getNextPageToken() != null) {
+                  while (! result.getNextPageToken().isEmpty()) {
```

Comment:
this https://developers.google.com/resources/api-libraries/documentation/genomics/v1/java/latest/com/google/api/services/genomics/model/ListBasesResponse.html says that getNextPageToken may return null